Support/Testing/Patches/Disable-broken-Concurrent-tests.patch

```diff
@@ -1,253 +1,12 @@
-diff --git a/test/functionalities/thread/concurrent_events/TestConcurrentEvents.py b/test/functionalities/thread/concurrent_events/TestConcurrentEvents.py
```

Comment:
The new patch disables the whole suite of Concurrent tests (so smaller because it doesn't do each one separately).